mod_bot/main.py

```diff
@@ -110,7 +108,13 @@ def add_minutes(hstart, mstart, mn):
                 message_to_send = 'Arène "{0}" inconnue\n'.format(gym_name)
                 isOk = False
                 should_delete = False
-                
+            elif type(gym_data) is list:
```

Comment:
I think it would be better to always have gym_data of type list (or None), with 0, 1 or 2+ elements and check its length instead of testing its type

Reply:
Agreed, you shouldn't test type when the language has static typing :D